Comp2025/src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -629,9 +641,9 @@ public double getElevatorPosition( )
 
   /****************************************************************************
    * 
-   * Return elevator length for stowed state
+   * Return elevator height for stowed state
    * 
-   * @return elevator stowed state length
+   * @return elevator stowed state height
    */
   public double getElevatorStowed( )
```

Comment:
This is also returning a "height" and should be getHeightStowed (as in:  get the height that the robot uses when it is stowed)